Tests/Command/TestCase.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Tests\Command;
+
+use Symfony\Component\Filesystem\Tests\FilesystemTestCase;
+
+if (class_exists('Symfony\Component\Filesystem\Tests\FilesystemTestCase')) {
```

Comment:
You don't have to do this trick anymore, form SF >=2.8 there is always this class.